package.json

```diff
@@ -40,6 +40,7 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "bencevans-jsdox": "^0.4.11",
```

Comment:
Now you did the same as that other dude publishing `node-sonos` to npm. Maybe next time you better use a [scoped package](https://docs.npmjs.com/misc/scope).

Reply:
I tried that https://github.com/bencevans/jsdox/commit/740b9082bae90802c9580c348d5cff4b0ac7b9ae but npm wouldn't let me publish. Complained that I had to signup for private packages :unamused:

Reply:
As the above link states:

> To publish a public scoped package, you must specify `--access public` with the initial publication. This will publish the package and set access to public as if you had run `npm access public` after publishing.

Reply:
and by the way why did you put it in dep section? Isn't that something only used by developers?

Reply:
aaaahhhh. WOOOOOOOPPSS! Double whammy :man_facepalming: PR coming...